src/applications/discharge-wizard/discharge-wizard-entry.jsx

```diff
@@ -1,6 +1,5 @@
 import '../../platform/polyfills';
 import './sass/discharge-wizard.scss';
-import '@department-of-veterans-affairs/formation/js/sidenav';
```

Comment:
We can't import these directly, they're not transpiled by Babel. They probably shouldn't even be in the published module.

They're also not necessary, as far as I can tell. The sidebar on the discharge wizard still works for me without it.